capellambse_context_diagrams/filters.py

```diff
@@ -68,10 +68,10 @@ def uuid_filter(obj: common.GenericElement, label: str | None = None) -> str:
     str, cabc.Callable[[common.GenericElement, str | None], str]
 ] = {
     EX_ITEMS: lambda obj, _: exchange_items(obj),
-    FEX_EX_ITEMS: lambda obj, _: exchange_name_and_items(obj),
-    FEX_OR_EX_ITEMS: lambda obj, _: exchange_items(obj)
-    if obj.exchange_items
-    else obj.name,
+    FEX_EX_ITEMS: exchange_name_and_items,
+    FEX_OR_EX_ITEMS: lambda obj, label: exchange_items(obj)
```

Comment:
Note that the signature of this lambda is different from the `exchange_name_and_items` function used above it: The second argument to the function is optional (and defaults to `None`), while in the lambda it's mandatory. Is this intended?

Reply:
### Idea
The basic idea of this label-adjuster registry is having filters which alter the label of Diagram-elements (for they only alter exchange labels). Combining/Adding multiple filters to a diagram sometimes leads to the need on passing already filtered labels on to the next filter to not lose this content. For e.g. having the `NO_UUID` and `FEX_EX_ITEMS` applied will lead to a label where the name of the exchange and its exchange items are displayed while also dropping any UUID from it. But also `EX_ITEMS`, `FEX_EX_ITEMS` and `FEX_OR_EX_ITEMS` filters are invariant. That means it makes no sense to apply a combination/multiple of them to a diagram... the last one will be taken. For me this is obvious if one understood the objectives of the filters,but maybe this should be documented on the filters page.

### Technical (Mypy)
For me this is correctly picked up by mypy. Maybe it is better to compare all filters from the registry: https://github.com/DSD-DBS/capellambse-context-diagrams/blob/b29be0992a39fbde087c1b9c41ceee6a31bc5200/capellambse_context_diagrams/filters.py#L67-L74

The label is regarded unlike in `EX_ITEMS`. Then on line 74 the label is taken if there were no `exchange_items` to begin with and if the label is not None.